Hunting Queries/Microsoft 365 Defender/Exfiltration/unusual-volume-of-file-sharing.yaml

```diff
@@ -0,0 +1,59 @@
+id: bafc1446-1cc4-4f6d-ad76-1250b8c3b60c
+name: Unusual volume of file deletion by user.
```

Comment:
I think names and descriptions need to be reversed with other one in PR as it does not match with the query.

Reply:
Changed the file names to reflect correct descriptions.